### DISCUSSION TITLE
QMLTreeView for Qt 5.12.8

### DISCUSSION THREAD

**Discussion by MikePooh** [Daguerreo/QMLTreeView#17] [Show and tell]
I've figured out that QMLTreeView is able to work over Qt 5.12.8 (which is default for Ubuntu:focal) with few number of changes. If you willing I can make a PR to new branch with that changes.

**Comment by Daguerreo** [comment]
Yes, I would appreciate it. Use the include guards to discern the version used.

**Comment by Daguerreo** [comment]
@MikePooh any chance to see that PR?

**Reply by MikePooh** [reply]
Sorry, switch my focus a little bit )
